corehq/ex-submodules/auditcare/views.py

```diff
@@ -114,31 +114,31 @@ def model_instance_history(request, model_name, model_uuid, *args, **kwargs):
     context['change_history'] = history_for_doc(obj)
     context['model'] = model_name
     context['model_uuid'] = model_uuid
-    return render_to_response('auditcare/model_instance_history.html', context)
+    return render(request, 'auditcare/model_instance_history.html', context)
 
 @login_required()
 @user_passes_test(lambda u: u.is_superuser)
 def single_model_history(request, model_name, *args, **kwargs):
     # it's for a particular model
-    context=RequestContext(request)
+    context = {}
     db = AccessAudit.get_db()
     vals = db.view('auditcare/model_actions_by_id', group=True, startkey=[model_name, ''], endkey=[model_name, 'z']).all()
     model_dict= dict((x['key'][1], x['value']) for x in vals)
     context['instances_dict']=model_dict
     context['model'] = model_name
-    return render_to_response('auditcare/single_model_changes.html', context)
+    return render(request, 'auditcare/single_model_changes.html', context)
+
 
 @login_required()
 @user_passes_test(lambda u: u.is_superuser)
 def model_histories(request, *args, **kwargs):
     """
     Looks at all the audit model histories and shows for a given model
     """
-    context=RequestContext(request)
     db = AccessAudit.get_db()
     vals = db.view('auditcare/model_actions_by_id', group=True, group_level=1).all()
     # do a dict comprehension here because we know all the keys in this reduce are unique
-    model_dict= dict((x['value'][0], x['value']) for x in vals)
-    context['model_dict']=model_dict
-    return render_to_response('auditcare/model_changes.html', context)
+    model_dict = dict((x['value'][0], x['value']) for x in vals)
+    context = {'model_dict': model_dict}
+    return render(request, 'auditcare/model_changes.html', context)
```

Comment:
wow, some old-school Django in this file

Reply:
tell me about it.. :)